backstory/src/main/java/com/google/sps/story/StoryManagerImpl.java

```diff
@@ -45,6 +45,15 @@
   /** requestFactory - Builds and facilitates authenticated post requests. */
   private StoryManagerRequestFactory requestFactory;
 
+  /** serviceUrls - URLs for each story generation container */
+  private String[] serviceUrls = {"https://backstory-text-gen-1-pdaqhmzgva-uc.a.run.app",
+      "https://backstory-text-gen-2-pdaqhmzgva-uc.a.run.app",
+      "https://backstory-text-gen-3-pdaqhmzgva-uc.a.run.app",
+      "https://backstory-text-gen-4-pdaqhmzgva-uc.a.run.app",
+      "https://backstory-text-gen-5-pdaqhmzgva-uc.a.run.app"};
+
+  private int selectedUrlIndex = 0;
```

Comment:
A comment here could be helpful (and technically it needs Javadocs even as a private field). It could be especially good to explain why you put 0 as its value (although I'm guessing that's just the default). Also, shouldn't setting it to 0 happen in the constructor, as this isn't a constant?

```suggestion
  /** the index of the current selected url in serviceUrls */
  private int selectedUrlIndex;
```

Then, instantiate it in the constructor.